packages/react-core/src/components/DragDrop/DragDrop.tsx

```diff
@@ -17,13 +18,18 @@ interface DragDropProps {
   children?: React.ReactNode;
   /** Callback for drag event. Return true to allow drag, false to disallow. */
   onDrag?: (source: DraggableItemPosition) => boolean;
+  /** Callback for drag event. */
```

Comment:
how is this intended to be used differently from `onDrag`. That clarity should be included in the description.